lib/nacre/concerns/resourceable.rb

```diff
@@ -10,12 +14,12 @@ def resource_class
 
   private
 
-  def format_service_name(name)
-    resource_class.to_s.gsub(/\ANacre::/, '').downcase
+  def format_resource_name(name)
```

Comment:
Unused method argument - `name`. If it's necessary, use `_` or `_name` as an argument name to indicate that it won't be used. You can also write as `format_resource_name(*)` if you want the method to accept any arguments but don't care about them.